### DISCUSSION TITLE
Volume control on an AOC U27N3C

### DISCUSSION THREAD

**Discussion by lzr1130** [MonitorControl/MonitorControl#1272] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

When I turn the volume down to minimum, the monitor is still very loud.
My monitor which is an AOC U27N3C can not control the volume by hardware.

### Steps to reproduce

When I connect the Mac to an AOC U27N2C and turn the volume down to minimum. The monitor still speaks very loud.
When I continue to turn the volume louder, the voice changes very slowly.

### Expected behavior

When I turn the volume down to minimum, the monitor's voice should be very small.

### Environment Information (please complete the following information)

```markdown
- macOS version: Ventura 13.0.1
- Mac model: MacBook Pro (13-inch, M1, 2020)
- MonitorControl version: 4.1.0
- Monitor(s): AOC U37N3C
- Apple Silicon/M1 (yes or no): yes
```

**Comment by waydabber** [comment]
This seems to be a monitor specific issue. Sadly not all display support DDC volume control properly. There is nothing much we can do about it, the display maker should address this problem with a firmware update (but they usually never do + such a thing, as most display does not have user upgradeable firmware). Sorry about this. :(
